### PR TITLE
Bratseth/reject cell typed tensor queries

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java [hunk 1]
```diff
@@ -33,12 +30,21 @@ public void testValidation() {
         new QueryValidator().search(new Query("?query=mystring:foo"), execution);
 
         try {
-            new QueryValidator().search(new Query("?query=mytensor:foo"), execution);
-            fail("Excpected validation error");
+            new QueryValidator().search(new Query("?query=sddocname%3Aproduct%20lfmModel25KeysV0%3A9%2A%20mytensor1%3A%3E0"), execution);
```
Comment (geirst) [inline container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java:33]: Consider simplifying query to only contain _mytensor1_.

File: container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java [hunk 2]
```diff
@@ -33,12 +30,21 @@ public void testValidation() {
...
+            fail("Expected validation error");
         }
         catch (IllegalArgumentException e) {
             // success
-            assertEquals("Cannot search 'mytensor': It is a tensor field", e.getMessage());
+            assertEquals("Cannot search 'mytensor1': It is a tensor field", e.getMessage());
+        }
+
+        try {
+            new QueryValidator().search(new Query("?query=sddocname%3Aproduct%20lfmModel25KeysV0%3A9%2A%20mytensor2%3A%3E0"), execution);
```
Comment (geirst) [inline container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java:42]: Consider simplifying query to only contain _mytensor2_.

**Review by geirst — APPROVED** [review]
👍

**Review by jonmv — APPROVED** [review]

File: container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java (hunk 2 above)
Comment (jonmv) [reply container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java]: 👍

File: container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java (hunk 1 above)
Comment (bratseth) [reply container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java]: done, thanks

File: container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java (hunk 2 above)
Comment (bratseth) [reply container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java]: done, thanks

Comment (jonmv) [reply container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java]: Reason for complicated query perhaps to verify nothing's lost?

**Pull request merged**